File: Java/FarmingTool.java
public class FarmingTool {
    private int idTool;
    private String farmingToolName;

    // Default constructor
    public FarmingTool() {
        this.idTool = 0;
        this.farmingToolName = "";
    }

    // Parameterized constructor
    public FarmingTool(int idTool, String farmingToolName) {
        this.idTool = idTool;
        this.farmingToolName = farmingToolName;
    }

    // Getter methods
    public int getIdTool() {
        return idTool;
    }

    public String getFarmingToolName() {
        return farmingToolName;
    }

    // Setter methods (tanpa setter untuk idTool)
    public void setFarmingToolName(String farmingToolName) {
        this.farmingToolName = farmingToolName;
    }

    // Display method
    public void displayTools(int indent) {
        String sp = " ".repeat(indent);
        System.out.println(sp + "Tool ID  : " + getIdTool());
        System.out.println(sp + "Tool Name: " + getFarmingToolName());
        System.out.println();
    }
}

File: Python/FarmingTool.py
class FarmingTool:
    def __init__(self, idTool: int = 0, farmingToolName: str = ""):
        self.idTool = idTool
        self.farmingToolName = farmingToolName

    # Getter methods
    def getIdTool(self) -> int:
        return self.idTool

    def getFarmingToolName(self) -> str:
        return self.farmingToolName

    # Setter methods (tanpa setter untuk idTool)
    def setFarmingToolName(self, farmingToolName: str):
        self.farmingToolName = farmingToolName

    # Display method
    def display_tools(self, indent: int = 0):
        sp = " " * indent
        print(f"{sp}Tool ID  : {self.getIdTool()}")
        print(f"{sp}Tool Name: {self.getFarmingToolName()}\n")

    # Destructor (opsional di Python, biasanya tidak perlu)
    def __del__(self):
        pass
